Mimir/Controllers/TableSheetsController.cs

```diff
@@ -12,23 +9,34 @@ namespace Mimir.Controllers;
 public class TableSheetsController(TableSheetsRepository tableSheetsRepository) : ControllerBase
 {
     [HttpGet("names")]
-    public async Task<string[]> GetSheetNames(
-        string network
-    )
+    public async Task<string[]> GetSheetNames(string network)
     {
         var sheetNames = tableSheetsRepository.GetSheetNames(network);
 
         return sheetNames;
     }
 
     [HttpGet("{sheetName}")]
-    public async Task<ContentResult> GetSheet(
+    public async Task<IActionResult> GetSheet(
         string network,
-        string sheetName
+        string sheetName,
+        string? format = "csv"
```

Comment:
How about using the `Accept` header instead of the `format` parameter? It's not clear how to do this well in ASP.NET. This is a comment that doesn't matter if it's reflected in the task schedule or later.

Reply:
Good points. i have finished this work
could you check it again?

Reply:
I left reviews for both!